spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/procedures/RemoveOrphanFilesProcedure.java

```diff
@@ -113,6 +123,19 @@ public StructType outputType() {
         action.executeDeleteWith(executorService(maxConcurrentDeletes, "remove-orphans"));
       }
 
+      if (tableWithActualFilePaths != null) {
+        Preconditions.checkArgument(spark().catalog().tableExists(tableWithActualFilePaths),
+                "actual_file_table `" + tableWithActualFilePaths + "` does not exist"
```

Comment:
This might not work if the `tableWithActualFilePaths` exists in another catalog than the current one.

We can probably try parsing it into Spark's `TableIdentifier`, but the below shows how it would look if the table _was_ in fact in another catalog.

```
boolean isTablePresent = sparkSession.catalog().tableExists("db_name","table_name")
```